README.md

```diff
@@ -46,6 +46,7 @@ lastlogin | Exposes the last time there was a login.
 megacli | Exposes RAID statistics from MegaCLI.
 ntp | Exposes time drift from an NTP server.
 runit | Exposes service status from [runit](http://smarden.org/runit/).
+tcpstat | Exposes tcp connection status from /proc/net/tcp and /proc/net/tcp6. (Warning, current version has performance issue in high load situation.)).
```

Comment:
Some minor things here, can we do:

"Exposes TCP connection status information from `/proc/net/tcp` and `/proc/net/tcp6`. (Warning: the current version has potential performance issues in high load situations.)"